### PR TITLE
Master - Removed strings from dummy template AAABase part 1

### PR DESCRIPTION
**Pull request by s7design**
Hi @mgruner 

This is the first part of removed AAABase dummy template. 

Regards
Zoran

### CODE REVIEW

File: Kernel/Output/HTML/Statistics/View.pm [hunk 1]
```diff
@@ -2043,18 +2043,28 @@ sub _TimeScaleBuildSelection {
...
+            Week     => Translatable('week(s)'),
+            Month    => Translatable('month(s)'),
+            Quarter  => Translatable('quarter(s)'),
+            HalfYear => Translatable('half-year(s)'),
+            Year     => Translatable('year(s)'),
         },
         Sort           => 'IndividualKey',
-        SortIndividual => [ 'Second', 'Minute', 'Hour', 'Day', 'Week', 'Month', 'Quarter', 'HalfYear', 'Year' ],
+        SortIndividual => [
+            Translatable('Second'),
```
Comment (mgruner) [inline Kernel/Output/HTML/Statistics/View.pm]: I think these are internal identifiers and do not have to be translated.

**Comment by mgruner** [comment]
The rest looks just fine.

**Comment by s7design** [comment]
Hi @mgruner 

Thanks for follow-up. I updated PR.

Regards
Zoran

**Comment by mgruner** [comment]
Excellent, thank you! I reverted one more SortIndividual line in Date.pm which refers to internal IDs.

**Pull request closed without merging**